app/javascript/components/__tests__/ProductivityBadFollowUp.test.js

```diff
@@ -22,15 +22,15 @@ describe('ProductivityBadFollowUp', () => {
   test('checking components for crashing', () => {
     render(
       <MemoryRouter>
-        <ProductivityBadFollowUp data={data} setData={setData} service={mockService} />
+        <ProductivityBadFollowUp data={data} setData={setData} saveDataToDb={() => {}} steps={[]} service={mockService} draft={false} />
       </MemoryRouter>
```

Comment:
Please investigate a way to remove duplication by putting the block into a variable. I expect it won't be too difficult.